e2e/framework/README.md

```diff
@@ -0,0 +1,23 @@
+# rook-test-framework
+ 
+Requirements:
+-   Docker 1.12.6 
```

Comment:
are the supported Docker versions a bit wider than just this one specific minor release?